src/components/hooks/useActivity.js

```diff
@@ -10,13 +10,13 @@ import {AdapterContext} from '../../components/';
  */
 export default function useActivity(activityID) {
   const [activity, setActivity] = useState({});
-  const adapter = useContext(AdapterContext);
+  const {activitiesAdapter} = useContext(AdapterContext);
```

Comment:
it's weird how eslint didn't complain about this when I added the rule.

Reply:
@akoushke I don't think this is part of the prefer destructuring rule.
It would error if you tried to do `const activitiesAdapter = adapter.activitiesAdapter`